### PR TITLE
Added forbidden hazardous goods to truck restrictions

### PR DESCRIPTION
**Pull request by dydome**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,16 +46,17 @@ Since the 1.1.0 version, you can calculate the route witch attributes of the rou
...
-  | weightPerAxle| int  | tons      | 0   | 1000|
+
+  | Property               | Type      | HumanType | Min | Max |
+  | ------                 | ----      | --------- | --- | --- |
+  | height                 | int       | meters    | 0   | 50  |
+  | width                  | int       | meters    | 0   | 50  |
+  | length                 | int       | meters    | 0   | 300 |
+  | limitedWeight          | int       | tons      | 0   | 1000|
+  | weightPerAxle          | int       | tons      | 0   | 1000|
+  | forbiddenHazardousGoods| [Available options](https://developer.here.com/documentation/routing/topics/resource-type-enumerations.html#resource-type-enumerations__enum-hazardous-good-type-type)  | ----      | --- | ---|
```
Comment (Daredzik) [inline README.md]: Change name to correct one

File: dist/lrm-here.js [hunk 2]
```diff
@@ -391,6 +392,14 @@ if (typeof module !== 'undefined' && module.exports) {
 				return _truckRestrictions;
 			}
 
+			options.truckRestriction['shippedHazardousGoods']
```
Comment (Daredzik) [inline dist/lrm-here.js]: What ?

File: README.md (hunk 1 above)
Comment (dydome) [reply README.md]: changed

File: dist/lrm-here.js (hunk 2 above)
Comment (dydome) [reply dist/lrm-here.js]: fixed

File: README.md [hunk 3]
```diff
@@ -46,16 +46,17 @@ Since the 1.1.0 version, you can calculate the route witch attributes of the rou
 
 ## TruckRestriction `truckRestriction`
 Since the 1.2.0 version, you can calculate the route witch attributes of the truck. To achieve this set `routeRestriction.vehicleType: 'truck'` and fill options under `truckRestriction` object using properties from below.
```
Comment (Daredzik) [inline README.md:48]: Add information about added new option since 1.3.0 version

File: README.md [hunk 4]
```diff
@@ -46,16 +46,17 @@ Since the 1.1.0 version, you can calculate the route witch attributes of the rou
...
-  | weightPerAxle| int  | tons      | 0   | 1000|
+
+  | Property               | Type      | HumanType | Min | Max |
+  | ------                 | ----      | --------- | --- | --- |
+  | height                 | int       | meters    | 0   | 50  |
+  | width                  | int       | meters    | 0   | 50  |
+  | length                 | int       | meters    | 0   | 300 |
+  | limitedWeight          | int       | tons      | 0   | 1000|
+  | weightPerAxle          | int       | tons      | 0   | 1000|
+  | shippedHazardousGoods| [Available options](https://developer.here.com/documentation/routing/topics/resource-type-enumerations.html#resource-type-enumerations__enum-hazardous-good-type-type)  | ----      | --- | ---|
```
Comment (Daredzik) [inline README.md]: Add `array` to type, before link to available options

Comment (dydome) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (dydome) [reply README.md:48]: added

**Pull request merged**